concierge/service/src/main/java/org/eclipse/ditto/concierge/service/common/DefaultEnforcementConfig.java

```diff
@@ -38,6 +38,7 @@
     private final int bufferSize;
     private final boolean globalLiveResponseDispatching;
     private final Set<String> specialLoggingInspectedNamespaces;
+    private final EntityCreationConfig restrictCreation;
```

Comment:
The added field `restrictCreation` is missing from:
* `equals()` & `hashCode()`
* `toString()`